apps/site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -12,46 +12,40 @@ This guide helps get you set up and introduces some of the key features of the g
 
 In practice, most block developers will not need to know the lower-level details of the specifications, as the libraries we provide implement them.
 
-## Choosing your approach
+## Creating a block
 
-We provide three templates which allow you to define the entry point for your block in different ways:
+1.  Move to a folder where you want to create your block.
+1.  Run `npx create-block-app@latest your-block-name`.
+1.  Switch to your new folder: `cd [your-block-name]`.
+1.  Run `yarn install && yarn dev` or `npm install && npm run dev`.
+1.  Open [http://localhost:63212](http://localhost:63212/) in your browser to see the starter template.
 
-- `custom-element`: create a block defined as a custom element (also known as Web Components).
-- `html`: create a block defined as an HTML file, with JavaScript added via `<script>` tags.
-- `react`: create a block defined as a React component.
+### I want to use a different technology
 
-To create a new block, run `npx create-block-app block-name --template template`, replacing `block-name` with the name to give your block, and `template` with one of the template names listed above.
+If you want to write blocks **using other technologies or frameworks**, we provide three templates to choose from which allow you to define the entry point for your block in different ways. To change the template, pass a parameter to the above command as follows: `npx create-block-app@latest your-block-name --template <TEMPLATE>`.
 
-### I want to use a different technology
+Currently the three templates are:
 
-If you want to write blocks **using other technologies or frameworks**, you have several options:
+1.  `react` - the default template which uses React and has a suite of helpful hooks you can use.
+1.  `custom-element` - create a block defined as a custom element (also known as Web Components), which is based on [Lit](https://lit.dev/). You can also use different approaches when constructing the element-i.e. use a custom element as a wrapper for anything else you like.
+1.  `html` - give up some of the ecosystem to roll-your-own, import and use additional libraries inside the `<script>` tag.
```

Comment:
I find "give up some of the ecosystem to roll-your-own" confusing. Which ecosystem am I giving up? React (the default)? The Block Protocol's? If the latter what does that mean as I write a Block Protocol block?

Reply:
```suggestion
1.  `html` - plain HTML and JavaScript. Import and use additional libraries inside the `<script>` tag.
```